STX.REST.RESTFulSense.Clients.Tests.Unit/Services/Foundations/HttpExchanges/HttpExchangeServiceTests.cs

```diff
@@ -255,7 +255,9 @@ private static dynamic CreateRandomHttpContent()
         private static dynamic CreateRandomHttpProperties(
             bool sendUrlParameters,
             bool sendRequestHeaders,
-            bool sendRequestContent)
+            bool sendRequestContent,
+            bool sendRequestContentHeaders,
+            string sendRequestHttpMethod)
```

Comment:
this is not a good variable name, i thought it was a bool but it´s a string, i think it should be just httpMethod.

Reply:
resolved